File: Modul 5/Soal 1/PRAK501-2410817210006-Andre_Cristian_Nathanael.py
def MaxBilangan(a, b, c, d): 
    nilai = a 
    if b > nilai: 
        nilai = b 
    if c > nilai: 
        nilai = c 
    if d > nilai: 
        nilai = d 
    return nilai 

a, b, c, d = map(int, input().split()) 
hasil = MaxBilangan(a, b, c, d) 
print(hasil) 

File: Modul 5/Soal 1/PRAK501-2410817210006-Andre_Cristian_Nathanael.c
#include <stdio.h>
int MaxBilangan(int a, int b, int c, int d){
    int nilai = a;
    if(b > nilai){
        nilai = b;
    }
    if(c > nilai){
        nilai = c;
    }
    if(d > nilai){
        nilai = d;
    }
    return nilai;
}
int main(){
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    return 0;
}